index.html

```diff
@@ -109,9 +128,7 @@ <h2>
         Introduction
       </h2>
       <p>
-        New types of mobile devices are appearing that have some sort of folding capabilities around the screen. Hence, 
-        when the device folds, it physically can be different form factor. The main interest in knowing the posture is to 
-        enable new user experiences with responsive design.
+        The device posture is the physical position in which a device holds which may be derived from sensors in addition to the angle. New types of mobile devices are appearing that have some sort of capabilities that allow them to change their posture. The most common types of devices are the ones that can fold (their screen or around their screen), allowing them to physically alter their form factor. The main interest in knowing the posture of a device is to enable new user experiences with responsive design.
```

Comment:
Please break lines... you can use tidy if you want - this is unreviewable on Github :-(

Reply:
I'll run it through tidy.